src/v2/content_digest.rs

```diff
@@ -117,25 +139,39 @@ mod tests {
     }
 
     #[test]
-    fn try_verify_succeeds_with_same_content() -> Fallible<()> {
+    fn verify_content_ok() -> Fallible<()> {
```

Comment:
```suggestion
    fn verify_succeeds_with_same_content() -> Fallible<()> {
```
so that we keep in line with `verify_fails_with_different_content`